database/nakadi/10_data/04_tables/01_event_type.sql

```diff
@@ -1,7 +1,10 @@
 CREATE TABLE IF NOT EXISTS zn_data.event_type (
   et_name varchar(255) NOT NULL PRIMARY KEY CHECK (et_name <> ''),
+  et_topic varchar(255) NOT NULL,
```

Comment:
I even considered having a separated table just to relate both. Event types and topics should be decoupled. I'm considering removing it from the EventType POJO and adding a new method to the repository in order to retrieve it directly from the database.